catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/GravitinoPaimonTable.java

```diff
@@ -71,12 +82,17 @@ public static GravitinoPaimonTable fromPaimonTable(Table table) {
         .withColumns(
             GravitinoPaimonColumn.fromPaimonRowType(table.rowType())
                 .toArray(new GravitinoPaimonColumn[0]))
+        .withPartitioning(toGravitinoPartitioning(table.partitionKeys()))
         .withComment(table.comment().orElse(null))
         .withProperties(table.options())
         .withAuditInfo(EMPTY)
         .build();
   }
 
+  public static Transform[] toGravitinoPartitioning(List<String> partitionKeys) {
```

Comment:
is it possible `partitionKeys` is null?

Reply:
if it is an unpartitioned paimon table, the partitionKeys will be an empty list, not null.

Reply:
ok